pctasks/run/pctasks/run/workflow/executor/remote.py

```diff
@@ -637,18 +626,32 @@ def execute_workflow(
 
             logger.info(f"Logging to: {log_uri}")
 
-            with WorkflowRunsContainer(
+            logger.info("Creating CosmosDB connections...")
+            # Create containers
+            wf_run_container = WorkflowRunsContainer(
                 WorkflowRunRecord, db=self.config.get_cosmosdb()
-            ) as container:
-                workflow_run = container.get(run_id, partition_key=run_id)
+            )
+
+            jp_container = WorkflowRunsContainer(
+                JobPartitionRunRecord, db=self.config.get_cosmosdb()
+            )
+
+            try:
+                wf_run_container.__enter__()
```

Comment:
Is there a reason to call `__enter__` and `__exit__` manually rather than using `with wf_run_container, jp_container`:?

Reply:
I actually didn't know you could use multiple context managers with a with, and didn't want to pay the double indent. Learned something new, updated in latest commit